plasma_framework/contracts/src/exits/payment/PaymentInFlightExitable.sol

```diff
@@ -28,7 +30,7 @@ contract PaymentInFlightExitable is
     using RLP for RLP.RLPItem;
     using UtxoPosLib for UtxoPosLib.UtxoPos;
 
-    uint8 constant public MAX_INPUT_NUM = 4;
+    uint constant public MAX_INPUT_NUM = 4;
```

Comment:
Minor: I might name this `uint256` instead of `uint`. I think in a for loop we don't really care how a `i` works as long as it has enough size and would like to read as little as possible. However, in global variables we might want to keep consistent with general code base to be explicit.

Reply:
done, but I went with uint8 here.